seacrowd/sea_datasets/sea_bench/sea_bench.py

```diff
@@ -0,0 +1,192 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{damonlpsg2023seallm,
+  author = {Xuan-Phi Nguyen*, Wenxuan Zhang*, Xin Li*, Mahani Aljunied*,
+            Qingyu Tan, Liying Cheng, Guanzheng Chen, Yue Deng, Sen Yang,
+            Chaoqun Liu, Hang Zhang, Lidong Bing},
+  title = {SeaLLMs - Large Language Models for Southeast Asia},
+  year = 2023,
+  Eprint = {arXiv:2312.00738},
+  url = {https://arxiv.org/pdf/2312.00738.pdf},
+}
+"""
+
+_DATASETNAME = "sea_bench"
+
+_DESCRIPTION = """\
+Sea-bench is a multilingual benchmark for assistant-style models annotated by native linguists
+covering 8 Southeast Asian languages. The linguists sourced such data by manually translating
+open-source English test sets, collecting real user questions from local forums and websites,
+collecting real math and reasoning questions from reputable sources, as well as writing test
+instructions and questions themselves. The Sea-bench test set contains 20 questions per task
+(5 tasks for 3 languages, 4 tasks for other 5 languages).
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/SeaLLMs/Sea-bench"
+
+_LANGUAGES = {"tgl": "tl", "khm": "km", "vie": "vi", "tha": "th", "lao": "lo", "mya": "my", "ind": "id", "zlm": "ms", "eng": "en"}
```

Comment:
Keep the `_LANGUAGES` containing list of the 3-code languages. Move the dict into new constant / variable.

Reply:
Will fix in the next commit. Thanks!